main.go

```diff
@@ -3,14 +3,24 @@ package mass_machine_type_transition
 import "os"
 
 func main() {
-	vmiInformer, err := getVmiInformer()
+	virtCli, err := getVirtCli()
 	if err != nil {
 		os.Exit(1)
 	}
-
-	if vmiInformer != nil {
+	
+	vmiInformer, err := getVmiInformer(virtCli)
+	if err != nil {
 		os.Exit(1)
 	}
-
+	
+	stopCh := make(chan struct{})
+	go vmiInformer.Run(stopCh)
```

Comment:
You can use `exitJob` for that purpose too

```suggestion
	exitJob = make(chan struct{})
	go vmiInformer.Run(exitJob)
```

Reply:
This will also initialize `exitJob` which is never initialized currently